src/docs/environment/index.mdx

```diff
@@ -89,23 +89,40 @@ computer is actually compiling Python!
 make setup-pyenv
 ```
 
+fish users will need to manually set some of the environment variables. This only needs to be done
+once.
+
+```shell {tabTitle: Fish}
+set -Ux PYENV_ROOT $HOME/.pyenv
+# fish >=3.2.0
+fish_add_path $PYENV_ROOT/bin
+# fish <3.2.0
+set -U fish_user_paths $PYENV_ROOT/bin $fish_user_paths
+```
+
 Once that's done, your shell needs to be reloaded. You can either reload it in-place, or close your terminal and start it
 again and cd into sentry. To reload it, run:
 
 ```shell
 exec "$SHELL"
-```
+````
```

Comment:
Do not make this change (3 single quotes is the right way).

Reply:
Thanks for catching this, looks like I copy-pasted a bad change to a couple of places 🙃